src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-use-external-keycloak.adoc

```diff
@@ -0,0 +1,77 @@
+// configuring-authorization
+
+[id="configuring-prod-short-to-use-external-keycloak_{context}"]
+= Configuring {prod-short} to use an external {identity-provider} installation
+
+By default, {prod-short} installation in multiuser mode includes the deployment of a dedicated {identity-provider} instance. However, using an external {identity-provider} is also possible. This option is useful when a user has an already existing {identity-provider} instance with already defined users.
+
+.Prerequisites
+
+ifeval::["{project-context}" == "che"]
+* This procedure is only applicable to {prod-short} installations done using the {prod-short} Operator. When using the `{prod-cli}` management tool and the `helm` installation mode to install {prod-short}, no supported method is available to use an external {identity-provider} instance.  
```

Comment:
This is unclear -- is it not supported when using `chectl` or when using `chectl` + the helm installation mode? If it's the former, we can drop "and the `helm` installation mode"